command.json

```diff
@@ -428,6 +432,176 @@
               "prompt": false
             }
           ]
+        },
+        {
+          "name": "bootstrap",
+          "description": "Bootstrap GKE cluster",
+          "arguments": [
+            {
+              "name": "environment",
+              "type": "string",
+              "description": "Environment (e.g. staging)"
+            },
+            {
+              "name": "project",
+              "type": "string",
+              "description": "GCloud project id (e.g. bedrock-foundation)",
+              "required": false,
+              "prompt": false
+            }
+          ]
+        },
+        {
+          "name": "provision",
+          "description": "Provision cluster on glcoud with Terraform (subcommands: <plan>, <apply>, <init> and <destroy>)",
```

Comment:
"GCloud"? maybe just a quick pass to align caps etc? I'm thinking maybe spell out "Google Cloud" as well but no major preference

Reply:
Will do.